content/en/platform/corda/5.0/developing-applications/api/api-crypto.md

```diff
@@ -101,4 +97,25 @@ dependencies {
 You must reference `net.corda:corda-crypto-extensions`.
 {{< /note >}}
 
-<!--For information about packaging a CorDapp, see the [development tutorial]().-->
\ No newline at end of file
+## Implementing Signature Schemes
+
+Corda supports the following `SignatureSpecs` (signature schemes) for creating `java.security.Signature` objects:
```

Comment:
>for creating `java.security.Signature` objects:

Not really sure this is correct. This implies that users can use `net.corda.v5.crypto.SignatureSpec` to create `java.security.Signature`? Which is not true. We are producers and consumers alone of  `net.corda.v5.crypto.SignatureSpec` API.